.gitignore

```diff
@@ -3,4 +3,5 @@ dist/
 dist-es5-module/
 npm-debug.log
 .DS_Store
-docs-preview/
+docs/
+!scripts/docs/
```

Comment:
`docs/` match every folder with that name, like for `node_modules/` so we need to exclude from gitignore `/scripts/docs` with `!` 👍